Programs/BRAT_CRB/Program.xml

```diff
@@ -32,6 +36,7 @@
       <Product id="BRAT" name="BRAT Models (HUC8)" folder="BRAT" url="" />
       <Product id="VBET" name="Valley Bottom Extraction Tool (VBET)" folder="VBET" url="" />
       <Product id="RSContext" name="Riverscapes Context (HUC8)" folder="RSContext" url="" />
+      <Product id="MODIS" name="MODIS (HUC4)" folder="MODIS" url="" />
```

Comment:
This `name` is what it will look like in the dropdown

Reply:
Sorry, I know we refer to this as MODIS within NAR, but can we use the name **LST** or Land Surface Temperature instead.

Someone could conceivably generate the same data from a different sensor or method.

Also, I was going to request that we abandon the "HUCx" suffixes in the product names. Can leave the legacy ones for now. But let's not add more products with these labels.